docs/source/use-cases.rst

```diff
@@ -296,14 +296,15 @@ URI.
 
 ::
 
-   blockchain://41941023680923e0fe4d74a34bdac8141f2540e3ae90623718e47d66d1ca4a2d/block/1e96de11320c83cca02e8b9caf3e489497e8e432befe5379f2f08599f8aecede
+   blockchain://41941023680923e0fe4d74a34bdac8141f2540e3ae90623718e47d66d1ca4a2d/block/5fd41fd8bbe935cdfc3bc89095b72dbd41c6308a4507e0f21059bf4a514995d4
 
 This URI defines the chain on which the ``SafeMathLib`` library was
 deployed. The first hash you see,
 ``41941023680923e0fe4d74a34bdac8141f2540e3ae90623718e47d66d1ca4a2d`` is
 the genesis block hash for the Ropsten test network. The later hash
-``1e96de11320c83cca02e8b9caf3e489497e8e432befe5379f2f08599f8aecede`` is
-the block hash for block number 168,238 from the Ropsten chain.
+``5fd41fd8bbe935cdfc3bc89095b72dbd41c6308a4507e0f21059bf4a514995d4`` is
+the block hash for block number 3,372,354 from the Ropsten chain, and should
+represent any block on which the contract is available.
```

Comment:
@pipermerriam Am I remembering this correctly?

Reply:
Correct, though the language is a little awkward.  Maybe...

> ... from the Ropsten chain.  This block number was arbitrarily chosen as one *shortly* after block on which the contract was deployed.  Any block number from this chain for which the contract is available at that block would be valid here.